src/components/APIDoc/SchemaDataView.tsx

```diff
@@ -56,15 +61,80 @@ const PropertyView:React.FunctionComponent<PropertyComponentProps> = ({propName,
       </FlexItem>
       <FlexItem>
         <TextContent>
-            <Text component={TextVariants.p}>
-              {propertyType}
-            </Text>
-          </TextContent>
+          <Text component={TextVariants.p}>
+            {propertyType}
+          </Text>
+        </TextContent>
+      </FlexItem>
+      <FlexItem>
+        {extraProps}
       </FlexItem>
     </Flex>
   )
 }
 
+
+interface ExtraPropertyViewProps {
+  propSchema: OpenAPIV3.SchemaObject;
+}
+const ExtraPropertyView:React.FunctionComponent<ExtraPropertyViewProps> = ({propSchema}) => {
+  let maxMin: string | undefined;
+  if (propSchema.maximum && propSchema.minimum) {
+    maxMin = (propSchema.exclusiveMinimum ? '>': '≥') + ` ${propSchema.minimum} and `
+    maxMin += (propSchema.exclusiveMaximum ? '<' : '≤') + ` ${propSchema.maximum}`
+  } else if (propSchema.maximum) {
+    maxMin = (propSchema.exclusiveMaximum ? '<' : '≤') + ` ${propSchema.maximum}`
+  } else if (propSchema.minimum) {
+    maxMin = (propSchema.exclusiveMinimum ? '>' : '≥') + ` ${propSchema.minimum}`
+  }
+
+  let maxMinChar: string | undefined;
+  if (propSchema.maxLength && propSchema.minLength) {
+    maxMinChar = `${propSchema.minLength} to ${propSchema.maxLength} chars`
+  } else if (propSchema.maxLength) {
+    maxMinChar = `max ${propSchema.maxLength} chars`
+  } else if (propSchema.minLength) {
+    maxMinChar = `max ${propSchema.minLength} chars`
+  }
+
+  let maxMinItems: string | undefined;
+  if (propSchema.maxItems && propSchema.minItems) {
+    maxMinItems = `${propSchema.minItems} and ${propSchema.maxItems} items`
+  } else if (propSchema.maxItems) {
+    maxMinItems = `max ${propSchema.maxItems} items`
+  } else if (propSchema.minItems) {
+    maxMinItems = `min ${propSchema.minItems} items`
+  }
+
+  let maxMinProps: string | undefined;
+  if (propSchema.maxProperties && propSchema.minProperties) {
+    maxMinProps = `${propSchema.minProperties} and ${propSchema.maxProperties} properties`
+  } else if (propSchema.maxProperties) {
+    maxMinProps = `max ${propSchema.maxProperties} properties`
+  } else if (propSchema.minItems) {
+    maxMinProps = `min ${propSchema.minProperties} properties`
+  }
+
+  return(
+    <TextContent>
```

Comment:
Could you use a `LabelGroup` instead? No special props are needed

https://patternfly-react.surge.sh/components/label-group